### PR TITLE
Avoid overriding files if they have the same content

### PR DESCRIPTION
**Pull request by reuveniy**

### CODE REVIEW

**Comment by reuveniy** [comment]
@schickling small change. would love to get this reviewed.

File: packages/graphqlgen/src/index.ts [hunk 1]
```diff
@@ -144,13 +155,22 @@ function writeResolversScaffolding(
   }
   const outputResolversDir = config['resolver-scaffolding']!.output
 
-  rimraf.sync(outputResolversDir)
+  const toBeCreatedFiles =
+    resolvers.map(f => path.join(outputResolversDir, f.path));
+
+  fs.readdirSync(outputResolversDir)
+    .map(f => path.join(outputResolversDir, f))
+    .filter(f => toBeCreatedFiles.indexOf(f) === -1)
```
Comment (timsuchanek) [inline packages/graphqlgen/src/index.ts]: Please use `includes` here. The rest of the PR looks good!

Comment (jasonkuhrt) [inline packages/graphqlgen/src/index.ts]:
```suggestion
    .filter(f => !toBeCreatedFiles.includes(f))    
```

**Comment by jasonkuhrt** [comment]
I did some refactoring a few days ago that will unfortunately require some conflict resolution. If I find time I'll do it myself but that might not happen.

**Comment by jasonkuhrt** [comment]
@reuveniy I will be working on this issue this weekend. I would like to give you credit for the original work, so will merely rebase and tweak this pr, then ship it. Hope that's ok with you.

**Pull request merged**